website/docs/d/oracle_gi_versions.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+subcategory: "Oracle"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_oracle_gi_versions"
+description: |-
+  Provides the list of GI (Grid Infrastructure) Versions.
+---
+
+# Data Source: azurerm_oracle_gi_versions
+
+This data source provides the list of GI Versions in Oracle Cloud Infrastructure Database service.
+
+Gets a list of supported GI versions.
+
+## Example Usage
+
+```hcl
+data "azurerm_oracle_gi_versions" "example" {
+  location = "West Europe"
+}
+
+output "id" {
+  value = data.azurerm_oracle_gi_versions.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `location` - (Required) The Azure Region to query for the GI Versions in.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `versions` - A list of valid Oracle Grid Infrastructure (GI) software versions.
```

Comment:
We already spell out the acronym above in the title so lets just refer to it as GI.
```suggestion
* `versions` - A list of valid GI software versions.
```